### PR TITLE
686 logging typos

### PR DESCRIPTION
**Pull request by kevinpowell1**

### CODE REVIEW

**Review by kevinpowell1 — COMMENTED** [review]
more typo fixes

File: pyproject.toml
Comment (bltravis) [inline pyproject.toml:1]: @kevinpowell1 Looks like this snuck in. Can we remove it, please?

File: src/folio_migration_tools/mapping_file_transformation/organization_mapper.py [hunk 1]
```diff
@@ -131,38 +131,26 @@ def set_up_reference_data_mapping(
             "categories",
         )
 
-        if address_categories_map:
-            self.address_categories_map = RefDataMapping(
-                *categories_shared_args, address_categories_map, "value", "CategoriesMapping"
-            )
-        else:
-            self.address_categories_map = None
+        self.address_categories_map = RefDataMapping(
```
Comment (bltravis) [inline src/folio_migration_tools/mapping_file_transformation/organization_mapper.py:134]: This doesn't seem like fixing typos. Was this included intentionally?

**Review by bltravis — CHANGES_REQUESTED** [review]
A couple questions inline.

**Comment by bltravis** [comment]
@kevinpowell1 Can you have a look at this?

**Comment by bltravis** [comment]
This was fixed in #804

**Pull request closed without merging**